test/server/banner.spec.js

```diff
@@ -70,6 +70,16 @@ describe("avatar", () => {
       .expect('Location', '/static/images/1px.png')
       .expect(302, done);
   });
+
+  it("handles the avatarHeight option", (done) => {
+    request(app)
+      .get('/yeoman/backers/0/avatar.jpg?avatarHeight=256')
+      .expect((res) => {
+        console.log("dimensions: ", sizeOf(res.body));
```

Comment:
Sure, I just copied the test from the `banner` – should I remove the `console.log` there too?